server/router/auth.ts

```diff
@@ -16,6 +16,7 @@ const validIDs = [
   '241748771473-e85o2d6heucd28loiq5aacese38ln4l4.apps.googleusercontent.com',
   '346199868830-dfi7n737u4g6ajl3ketot11d1m3n1sr3.apps.googleusercontent.com',
   '322014396101-8u88pc3q00v6dre4doa64psr9349bhum.apps.googleusercontent.com',
+  '407408718192.apps.googleusercontent.com',
```

Comment:
What is this?

Reply:
oops that was a client id i was trying to test authentication with